aztk/node_scripts/scheduling/common.py

```diff
@@ -0,0 +1,101 @@
+import datetime
+import os
+
+import azure.batch.models as batch_models
+import azure.storage.blob as blob
+import requests
+import yaml
+
+from aztk.node_scripts.core import config
+from aztk.node_scripts.scheduling import scheduling_target
+
+
+def load_application(application_file_path):
+    """
+        Read and parse the application from file
+    """
+    with open(application_file_path, encoding="UTF-8") as f:
+        application = yaml.load(f)
+    return application
+
+
+def upload_log(blob_client, application):
+    """
+        upload output.log to storage account
+    """
+    log_file = os.path.join(os.environ["AZ_BATCH_TASK_WORKING_DIR"], os.environ["SPARK_SUBMIT_LOGS_FILE"])
```

Comment:
can those environment variables be read from a constant instead?

Reply:
there is quite a few other below too. Believe that would make it less likely to have a typo error